src/NuGet.Core/NuGet.Configuration/PackageNamespaces/PackageSourceSection.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Configuration
+{
+    internal class PackageSourceSection
+    {
+        private readonly string _packageSourceKey;
+
+        public string GetPackageSourceKey()
+        {
+            return _packageSourceKey;
+        }
+
+        private readonly string[] _nameSpaceIds;
```

Comment:
Why not use auto property instead? 

Example: `public string PackageSourceKey { get; }`

Reply:
Ok. Removing this type.